packages/api/src/platforms/vtex/clients/commerce/index.ts

```diff
@@ -364,5 +374,48 @@ export const VtexCommerce = (
         { storeCookies }
       )
     },
+    rating: (productId: string): Promise<ProductRating> => {
+      return fetchAPI(
+        `${base}/${REVIEWS_AND_RATINGS_API_PATH}/rating/${productId}`,
+        undefined,
+        { storeCookies }
+      )
+    },
+    reviews: {
+      create: (input: CreateProductReviewInput): Promise<string> => {
+        return fetchAPI(
+          `${base}/${REVIEWS_AND_RATINGS_API_PATH}/reviews`,
+          {
+            ...BASE_INIT,
+            body: JSON.stringify(input),
+            method: 'POST',
+          },
+          { storeCookies }
+        )
+      },
+      list: ({
+        orderBy,
+        orderWay,
+        ...partialInput
+      }: ProductReviewsInput): Promise<ProductReviewsResult> => {
+        const formattedInput: Record<string, string> = adaptObject<string>(
```

Comment:
```suggestion
        const formattedInput = adaptObject<string>(
```
The `adaptObject` already return an `Record<string, string>` in this case.